src/stories/components/DataCard/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Data Card
+
+.data-card {
+    border-radius: 5px;
```

Comment:
Nitpick-y design thing, can you change this to be `20px`? Thanks!

Reply:
Was wondering if this should be a certain value, thanks!